bedrock/firefox/views.py

```diff
@@ -505,11 +505,11 @@ def download_thanks(request):
     elif locale == 'de' and experience == 'berlin':
         template = 'firefox/new/berlin/scene2.html'
     elif locale == 'en-US':
-        if experience == 'portland':
+        if experience == 'portland' or experience == 'forgood':
```

Comment:
Would be better as `experience in ['portland', 'forgood']`.

My brain was tired and distracted when I sent you in the `or` direction. Apologies. 😞